cloudbuild-publish.yaml

```diff
@@ -2,12 +2,23 @@ steps:
   - name: 'gcr.io/cloud-builders/gcloud'
     script: |
       #!/usr/bin/env bash
+      gcloud secrets versions access latest --secret=github-token-access --format='get(payload.data)' | tr '_-' '/+' | base64 -d > token.txt
```

Comment:
It would be good to extract these clunky bits of bash to a shared script somewhere, and make it obvious why each piped stage is done.

Debugging issues with this in the future will be a huge pain without doing that, and we'd likely have drift between the commands!

Reply:
I will create a separate ticket for this.